filebeat/tests/system/config/filebeat.yml.j2

```diff
@@ -132,11 +133,16 @@ processors:
 # Multiple outputs may be used.
 
 #------------------------------- File output ----------------------------------
+{%- if logstash %}
```

Comment:
not that we have enabled: true again, we probably don't need to full block comments ;-)